.github/workflows/setup_python_conda.yml

```diff
@@ -0,0 +1,35 @@
+name: set up python environment
+on: push
+
+jobs:
+  setEnv:
+    name: set up python conda environment
+    runs-on: macos-10.15
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.9.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - uses: s-weigand/setup-conda@v1
+        with:
+          activate-conda: true
+      - name: install Python dependencies
+        run: |
+          conda create -n manyFEMS
+          conda install numpy
+          conda install xlrd==1.2.0
```

Comment:
Please could you create a conda environment.yml file instead of specifying the dependencies here? (That will make it easier for other users to create a matching conda environment.)

Reply:
accepted and corrected.